app/modules/incident/incident_helper.py

```diff
@@ -513,17 +526,63 @@ def handle_update_status_command(
         "Reviewed",
         "Closed",
     ]
-
     if status not in valid_statuses:
-        respond("Invalid status. Valid statuses are: " + ", ".join(valid_statuses))
+        respond(
+            "A valid status must be used with this command:\n"
+            + ", ".join(valid_statuses)
+        )
+        return
+    incidents = incident_folder.lookup_incident("channel_id", body["channel_id"])
+    if not incidents:
+        respond(
+            "No incident found for this channel. Will not update status in DB record."
+        )
+        return
     else:
-        respond(f"Updating incident status to {status}...")
-        incident_status.update_status(
-            client,
-            ack,
-            respond,
-            status,
-            body["channel_id"],
-            body["channel_name"],
-            body["user_id"],
+        if len(incidents) > 1:
```

Comment:
Would that even be possible to have more than 1 incident in a channel?

Reply:
Not supposed to but there could be an instance where someone updates the channel_id field manually with an existing value.